index.html

```diff
@@ -5,27 +5,326 @@
+                    <div class="content">
+                        <h3> lorem ipsum</h3>
+                        <p> lorem ipsum dolor</p>
+                    </div>
+                </div>
                 <div></div>
-                <div></div>
-                <a href=""></a>
             </div>
-        </header>
-        <section>
-            <section class="left-column"></section>
-            <section class="right-column"></section>
-        </section>
+        </div>
+        <div class="section">
+            <div class="h1-title">
+                <div>
+                    <h3>Don´t miss the lastest</h3>
+                    <h3>articles</h3>
+                </div>
+                <div class="float-button"></div>
+            </div>
+            <div class="col">
+                <div>
+                    <h3>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. saepe
+                    </h3>
+                </div>
+            </div>
+            <div class="col">
+                <div>
+                    <h3>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. saepe
+                    </h3>
+                </div>
+            </div>
+            <div class="col sm-col">
+                <div>
+                    <h3>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. saepe
+                    </h3>
+                </div>
+            </div>
+            <div class="col sm-col">
+                <div>
+                    <h3>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. saepe
+                    </h3>
+                </div>
+            </div>
+        </div>
+        
+    </div>
+    <div class="row last-bold">
+        <div class="col-1-4">
+            <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit, est sequi. Voluptate maxime optio ipsa consequuntur. Ut architecto ipsam molestias unde, recusandae quaerat fuga nemo distinctio sapiente natus accusantium pariatur!</p>
+        </div>
+        <div class="col-1-4">
+            <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit, est sequi. Voluptate maxime optio ipsa consequuntur. Ut architecto ipsam molestias unde, recusandae quaerat fuga nemo distinctio sapiente natus accusantium pariatur!</p>
+        </div>
+        <div class="col-1-4">
+            <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit, est sequi. Voluptate maxime optio ipsa consequuntur. Ut architecto ipsam molestias unde, recusandae quaerat fuga nemo distinctio sapiente natus accusantium pariatur!</p>
+        </div>
+        <div class="col-1-4">
+            <div>
```

Comment:
Your page must have a section with 3 articles of 33% width each. Delete one of these columns and give the 3 articles the same width.